src/validation.h

```diff
@@ -508,6 +508,24 @@ class BlockManager {
 /** Global variable that points to the active CCoinsView (protected by cs_main) */
 extern std::unique_ptr<CCoinsViewCache> pcoinsTip;
 
+/**
+ * A convenience class for constructing the CCoinsView* hierarchy used
+ * to facilitate access to the UTXO set.
+ */
+class CoinsViews {
+
+public:
+    std::unique_ptr<CCoinsViewDB> m_dbview;
```

Comment:
nit : I know these fields weren't documented as global, but maybe add comment explaining the aforementioned hierarchy between them